provider/kubernetes_test.go

```diff
@@ -1543,6 +1612,10 @@ func (c clientMock) GetIngresses(namespaces k8s.Namespaces) []*v1beta1.Ingress {
 }
 
 func (c clientMock) GetService(namespace, name string) (*v1.Service, bool, error) {
+	if c.apiServiceError {
+		return &v1.Service{}, true, c.err
```

Comment:
Conventionally, if an error occurs in a function returning multiple values, the non-`error` parameters are set to their default values. Thus, I'd use `false` for the boolean return value.

Slightly related/unrelated: I don't know why we return `true` in line 1624 when we seemingly can't find the service.